samcli/commands/local/invoke/cli.py

```diff
@@ -14,14 +14,17 @@
 
 HELP_TEXT = """
 You can use this command to execute your function in a Lambda-like environment locally.
-You can pass in the event body via stdin or by using the -e (--event) parameter.
-Logs from the Lambda function will be output via stdout.\n
+You can pass in an event body using the -e (--event) parameter.
+Logs from the Lambda function will be written to stdout.\n
+\b
+Invoking a Lambda function without an input event
+$ sam local invoke "HelloWorldFunction"\n
 \b
 Invoking a Lambda function using an event file
 $ sam local invoke "HelloWorldFunction" -e event.json\n
 \b
 Invoking a Lambda function using input from stdin
-$ echo '{"message": "Hey, are you there?" }' | sam local invoke "HelloWorldFunction" \n
+$ echo '{"message": "Hey, are you there?" }' | sam local invoke "HelloWorldFunction" --event - \n
```

Comment:
does `--event -` automatically trigger reading from stdin?

Reply:
Yes. Pretty sure it is standard in unix.

http://click.palletsprojects.com/en/5.x/arguments/#file-arguments

Reply:
It does, I've tested this.